code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -146,6 +150,11 @@
 
 /datum/reagent/dermaline/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
 	M.heal_organ_damage(0, 12 * removed)
+	M.brute_mod = M.species.brute_mod += 2.3
```

Comment:
```suggestion
	M.brute_mod = M.species.brute_mod += 2.3
```

You're modifying the brute mod of the species here. This is baad. You probably want `M.species.brute_mod + 2.3`